### DISCUSSION TITLE
Automations fail on empty response

### DISCUSSION THREAD

**Discussion by silkylucas** [Budibase/budibase#6357] [Help]
Hi, 
I have been trying to use the automation feature to send reminder emails, by running an external data connector. However, the page gets stuck when i try to test it if the response for some reason is empty. Ideally for me this should just fail gracefully, but now I am trying to use the conditions to stop the automation on an empty response. Any suggestions?

**Comment by melohagan** [comment]
Hey @silkylucas 

You could try this:
![Screenshot 2022-06-16 at 15 56 41](https://user-images.githubusercontent.com/101575380/174098888-830f2197-e248-4370-9f56-3c65ae1b4d3d.png)

That would halt the automation at that point if the response is empty.

There's also a docs page here similar to what you're describing: https://docs.budibase.com/docs/conditions-1

**Reply by silkylucas** [reply]
Hi @melohagan!
Thank you for the suggestion! 

![image](https://user-images.githubusercontent.com/71893030/174106128-612810fa-37a6-424a-830d-7296194d1093.png)

However I still get stuck while testing:

![image](https://user-images.githubusercontent.com/71893030/174106252-c5a71076-ae04-4c4e-998d-d5fc4b85dd9f.png)
I am connecting to my own Postgres Database, could the error maybe stem from there?

**Reply by melohagan** [reply]
Possibly, there was also a recent bug fix around the external data connector. If you look at the output tab for the EDC, it should look roughly like this:
![Screenshot 2022-06-16 at 16 40 45](https://user-images.githubusercontent.com/101575380/174109368-69a4c748-7986-4267-9a74-db52cd38a96d.png)

If you're finding that the data is returned in the **info** section, then you'll need to upgrade to version **1.0.206**
